tools/reports/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/Kong/mesh-perf/tools/reports
```

Comment:
I expect this repo to have a couple more `main.go` tools and to be honest, I don't think we should create `go.mod` file for each new binary. It's way easier to maintain the project when there is only one `go.mod` in the root directory. 

Also, that's what we're doing in Kuma – there are plenty `main.go` and only 1 `go.mod`.

Reply:
I'm not sure it's totally comparable, this is basically an alternative to a GitHub action or bash script rather than a part of a larger project or application. Kind of unimportant that it's also written in go. It really depends on what interface is shared between the binaries and the compatible guarantees, if any. But if you prefer I can share the go.mod